learn-arai/app/components/icons/apple.tsx

```diff
@@ -0,0 +1,34 @@
+export const AppleIcon = () => {
```

Comment:
Is there any reasons to use arrow function instead of regular function?
If there is only one export, consider default export this function